docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
+- Django >= 1.7.10, < 1.8
+- Translate Toolkit >= 1.13.0
+- Python >= 2.7, < 3.0
+- Redis >= 2.8.4
+- Django transaction hooks
+- Unix-based operating system.
+
+
+Major Changes
+=============
+
+- Pulled latest translations.
+- MyISAM support has been dropped.
```

Comment:
This was dropped in practical terms in the 2.7.0 timeline, it wasn't documented perhaps.